src/libclipper/src/redis.cpp

```diff
@@ -438,6 +449,22 @@ bool add_application(redox::Redox& redis, const std::string& appname,
   }
 }
 
+bool add_model_links(redox::Redox& redis, const std::string& appname,
+                     const std::vector<std::string>& model_names) {
+  if (send_cmd_no_reply<string>(
+          redis, {"SELECT", std::to_string(REDIS_APP_MODEL_LINKS_DB_NUM)})) {
+    for (auto model_name : model_names) {
```

Comment:
Resolved. This function is not on any critical paths, so making a few extra database queries will not have any significant impact on performance.